### DISCUSSION TITLE
Best streak even if i select a fail

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#1199] [Help & FAQ] [answered]
Hi,

i have a daily habit. If i select, that i failed (did not do the habit), the app still counts that in for a streak.

In this example, my best streak should be 3, not 7, right?

![Screenshot_20211125-103049](https://user-images.githubusercontent.com/95022529/143416187-e966c021-82e8-42d6-a427-a09cfd6f3e86.png)

**Comment by hiqua** [comment, marked as answer]
You selected that you skipped the days, not that you failed it. If you go to your main screen, you should see `-` which means skipped, failed would be a `x`. If you don't want to be able to skip days, you can disable that in the settings.

**Reply by ghost** [reply]
Thank you, that helped me!
